aws-android-sdk-pinpoint/src/test/java/com/amazonaws/mobileconnectors/pinpoint/targeting/notification/PinpointNotificationActivityTests.java

```diff
@@ -2,43 +2,47 @@
 
 import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyZeroInteractions;
 
-import android.content.Context;
-import android.content.Intent;
 import android.os.Bundle;
 
+import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.Mockito;
+import org.robolectric.Robolectric;
 import org.robolectric.RobolectricTestRunner;
 import org.robolectric.annotation.Config;
+import org.robolectric.android.controller.ActivityController;
 
 @RunWith(RobolectricTestRunner.class)
 @Config(manifest = Config.NONE, sdk=23)
-public class PinpointNotificationReceiverTests {
+public class PinpointNotificationActivityTests {
 
     private NotificationClient notificationClient;
-
-    private PinpointNotificationReceiver receiver;
-
-    private Intent intent;
-
-    private Context context;
+    private ActivityController<PinpointNotificationActivity> activityController;
 
     @Before
     public void setup() {
+        PinpointNotificationActivity.setNotificationClient((NotificationClient) null);
```

Comment:
why are we setting this to `null` here before immediately overriding it with a mock?

Reply:
We could remove here. It was just an effort to make sure we first started all tests in a clean slate (ex: this static variable being wiped out), before moving on to additional setup configuration.